src/Symfony/Bridge/Doctrine/Tests/ContainerAwareEventManagerTest.php

```diff
@@ -20,7 +20,10 @@ class ContainerAwareEventManagerTest extends TestCase
     private $container;
     private $evm;
 
-    protected function setUp()
+    /**
+     * @before
+     */
+    protected function before(): void
```

Comment:
`void` cannot be used on 3.4, because PHP 5.5 is supported.

Reply:
You're right, I didn't remember the support for PHP 5.5.